slf4cats-impl/src/main/scala/slf4cats/encoders/jackson/package.scala

```diff
@@ -0,0 +1,14 @@
+package slf4cats.encoders
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import slf4cats.api.LogEncoder
+
+package object jackson {
+  val jackson = new ObjectMapper()
+
+  implicit def encoder[A]: LogEncoder[A] = {
+    jackson.registerModule(DefaultScalaModule)
+    jackson.writeValueAsString
+  }
+}

```

Comment:
```suggestion
package object jackson {
  implicit val encoder: LogEncoder[Any] = {
    val jackson = new ObjectMapper()
    jackson.registerModule(DefaultScalaModule)
    jackson.writeValueAsString
  }
}
```
I think the `jackson` object shouldn't be exposed -- it's mutable, and so it could "magically" change things when you don't want/expect
also, `new ObjectMapper()` and `jackson.registerModule` should be executed only once

Reply:
application of the diff didn't work as you've probably intended :smile_cat: